docs/ipac/irsa/most/most.rst

```diff
@@ -0,0 +1,449 @@
+
+Depending on what input mode Most is being queried as, the list of required
+parameters will change.
+
+Output Modes
+============
+
+The service accepts several different output modes:
+ - ``Regular`` - default
+ - ``Full``
+ - ``Brief``
+ - ``VOTable``
+ - ``Gator``
+
+The :meth:`~astroquery.ipac.irsa.most.MostClass.query_object` method will
+return different objects depending on the specified output mode.
+
+=========================== ==================================================
+Output mode                 Returned object
+=========================== ==================================================
+``"Regular"`` or ``"Full"`` Dictionary containing ``results``, ``metadata``
+                            and ``region`` keys. Optionally also
+                            ``fits_tarball`` and ``region_tarball``.
+``"Brief"`` or ``"Gator"``  :class:`~astropy.table.Table` object.
+``"VOTable"``               :class:`~astropy.io.votable.tree.VOTableFile`
+                            object.
+=========================== ==================================================
+
+.. note::
+    The difference between ``Regular`` and ``Full`` output mode is non-existant
+    as the returned data in both cases is identical, as the figures created in
+    ``Full`` mode are not downloaded. The difference between the two modes are
+    mainly visible in presentation of the data when Most is used via their
+    online interface. It is, therefore, recommended ``Regular`` because the
+    query will complete significantly faster.
+
+Regular and Full
+________________
+
+In ``"Regular"`` or ``"Full"`` output mode :meth:`~astroquery.ipac.irsa.most.MostClass.query_object`
+returns a dictionary containing ``results``, ``metadata`` and ``region`` keys.
+
+The ``results`` key contains a table that maps to the results table returned by
+Most service titled ``Images with a Matched Object Position``. The table
+contains the following columns:
+
+============   ===============================================================
+Column 	       Description
+============   ===============================================================
+Image_ID       A unique identifier for the image data, not necessarily the
+               image file name.
+date_obs       Date (UTC) of the observation.
+time_obs       Time (UTC) of the midpoint of the observation.
+mjd_obs        Modified Julian Date (days) of the midpoint of the observation.
+ra_obs         Ephemeris of the object's right ascension (deg, J2000).
+dec_obs        Ephemeris of the object's declination (deg, J2000).
+sun_dist       Heliocentric distance of the object (AU).
+geo_dist       Geocentric distance of the object (AU).
+dist_ctr       Projected distance  from the center of the image (deg).
+phase          Sun-Object-earth angle (deg).
+vmag           Estimate of visual magnitude (from Horizons).
+image_url      Links to download or view the data.
+postcard_url   ??? usually ``null``
```

Comment:
I wanted to leave a comment to point out that I added some most basic error checks to the query - errors are re-raised but when no results are found I return `None`. Could not make out the standard expected behavior from other classes. 

Additionally, I still don't know what a `postcard_url` is - usually it's just `null`. This should be clarified before merge I'd think.